<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenSetting</class>
 <widget class="QWidget" name="ScreenSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleAspectRation">
     <property name="text">
      <string>播放比例</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="defaultAspectRatioButton">
        <property name="text">
         <string>默认</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="aspectRation43Button">
        <property name="text">
         <string>4:3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="aspectRation169Button">
        <property name="text">
         <string>16:9</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="aspectRationFullButton">
        <property name="text">
         <string>填充</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="titleRotation">
     <property name="text">
      <string>旋转</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="clockwiseRotationButton">
       <property name="text">
        <string>顺时针旋转</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="anticlockwiseRotationButton">
       <property name="text">
        <string>逆时针旋转</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="horizontalFilpButton">
       <property name="text">
        <string>水平翻转</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="verticallyFilpButton">
       <property name="text">
        <string>竖直翻转</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="titleImageQualityEnhancement">
     <property name="text">
      <string>画质增强</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="imageQualityEnhancementButton">
     <property name="text">
      <string>开启画质增强</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>78</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
